examples/bq-email-exports/README.md

```diff
@@ -0,0 +1,50 @@
+# Automated BigQuery Exports via Email
+
+This serverless solution enables users to regularly send BigQuery export results via email. The end users will get a scheduled email with a link to a Google Cloud Storage [signed URL](https://cloud.google.com/storage/docs/access-control/signed-urls), from which they can view query results as a JSON, CSV, or Avro file.
+
+The functional steps are listed here:
+
+**Cloud Scheduler:** A [Cloud Scheduler](https://cloud.google.com/scheduler) job defines the time frame between periodic email exports.
```

Comment:
Not to move the goal post again and perhaps you've already ruled this out (or built this a while ago), but why not use BigQuery Data transfer service Scheduled Queries (native to BQ no Cloud Scheduler dependency)?https://cloud.google.com/bigquery/docs/scheduling-queries#configuration_options

These can be setup w/ Pub/Sub notifications out of the box.
https://cloud.google.com/bigquery-transfer/docs/transfer-run-notifications

This would get rid of Cloud Scheduler, Pub/Sub topic #1 and Cloud Function #1 and instead use BQ scheduled queries to write directly to Pub/Sub topic. Also there will be no need for the logging sink / filter because we can handle this more explicitly.

I think this is a much cleaner solution (and one that will eliminate your VPC-SC caveat!)

Reply:
Definitely! I have implemented this in the most recent commit.